### ISSUE TITLE
Dataset Download Link is Broken

### ISSUE THREAD

**Issue by dianzizs** [zexiJia/EvSegFormer#1]
Thank you for sharing this project. While going through your project, I noticed that the download link provided for the ground truth of dataset ddd17 is no longer working. Would you be able to provide an updated download link, so we can continue using the dataset in your project?

Here is the broken download link: https://drive.google.com/open?id=1Ug6iZc7WYQWCklxwcemCeyw3CPyuuxJf

Looking forward to your response, thank you very much!

Best regards

**Comment by zexiJia** [comment]
I apologize, but we no longer have access to the dataset DDD17 provided by EvSegNet. It appears that they have recently removed the related project from GitHub, and the data link is no longer valid. Therefore, we are unable to provide the dataset as we do not have the necessary permissions or copyrights.

As an alternative, you can refer to the approach mentioned in our paper, which involves training a segmentation network based on APS photos and providing labels. You can use other available datasets to train your network instead.

**Comment by dianzizs** [comment]
Here is an updated link for ddd17 that I collected from another source, I hope it will be helpful to you. The link is: [https://pan.baidu.com/s/172PVZpwxwOyTC-w-ok_fIg?pwd=j3l2]

In addition, regarding the method mentioned in the paper, I noticed that the link for the weights of the labels-generating model (https://drive.google.com/drive/folders/1NjTGAoSCpYw_l89l1BdulJi0B_qIIiin?usp=sharing) seems to be broken as well. May I kindly request a new link if possible? Thank you very much!
